pkg/storage/disk.go

```diff
@@ -145,10 +146,35 @@ func (s *Module) DiskWrite(name string, image string) error {
 		return fmt.Errorf("image size is bigger than disk")
 	}
 
-	_, err = io.Copy(file, source)
-	if err != nil {
+	// writing the image concurrently to speedup the previous sequential write.
+	// the sequential write is slow because the data source is from the remote server.
+	var (
+		// use errgroup because there is no point in continuing if one of the goroutines failed
+		group               = new(errgroup.Group)
+		concurrentNum int   = 5
```

Comment:
change to numWorkers

Reply:
how was the 5 choosen?

Reply:
> how was the 5 choosen?

see https://github.com/threefoldtech/zos/pull/2395#issuecomment-2285301322

Reply:
> change to numWorkers

done